TombEngine/Game/effects/weather.cpp

```diff
@@ -9,44 +9,24 @@
 #include "Game/effects/tomb4fx.h"
 #include "Game/savegame.h"
 #include "Game/Setup.h"
-#include "Math/Random.h"
+#include "Math.h"
+#include "Objects/game_object_ids.h"
 #include "Sound/sound.h"
-#include "Specific/level.h"
 #include "Scripting/Include/ScriptInterfaceLevel.h"
+#include "Specific/level.h"
 
 using namespace TEN::Collision::Point;
 using namespace TEN::Effects::Ripple;
-using namespace TEN::Math::Random;
+using namespace TEN::Math;
 
 namespace TEN::Effects::Environment 
 {
-	constexpr auto WEATHER_PARTICLES_SPAWN_DENSITY = 32;
```

Comment:
Why these constants were moved to .h file, if their scope is only this particular module? There is no need to expose them on global scale.

Reply:
It was for having all constants in the same place :-)